server/views/pages/activities/unlock-list/planned-events.njk

```diff
@@ -147,6 +147,34 @@
                     ]
                 }) }}
 
+                {% if activityCategories %}
+                    {% set activityCategoriesFilters = [] %}
+                    {% for category in activityCategories %}
+                        {% set activityCategoriesFilters = (activityCategoriesFilters.push({
+                            value: category.code,
+                            text: category.name,
+                            checked: category.code in session.unlockListJourney.activityCategoriesFilters
+                        }), activityCategoriesFilters) %}
+                    {% endfor %}
+
+                    {{ govukCheckboxes({
+                        idPrefix: 'activityCategoriesFilters',
+                        name: 'activityCategoriesFilters',
+                        classes: "govuk-checkboxes--small",
+                        fieldset: {
+                            legend: {
+                                text: 'Activity Categories',
```

Comment:
Just checking - should it be "Active category" ?

Reply:
It should, but I can change this when I put my PR in (it will change this file quite a lot)